ansys_optical_automation/application/rotation_study.py

```diff
@@ -0,0 +1,70 @@
+import math
+import time
+
+
+def rotation_study(theta_inc, phi_inc, source, simulation):
+    """
+    function to apply rotation light source study and export simulation
```

Comment:
Ambient source not general source-> test for ambient source is missing can lead to alot of confusion